### ISSUE TITLE
I am not able to view any course under Courses tab please help  #coursenotfound

### ISSUE THREAD

**Issue by vijay6696** [FaisalUmair/udemy-downloader-gui#596]
![image](https://user-images.githubusercontent.com/84227463/118356150-8d461d80-b591-11eb-9a6f-26b57ff2e42b.png)

**Comment by vijay6696** [comment]
hi @FaisalUmair can you help

**Comment by Mr-WhiteMan** [comment]
Did u buy any course?

**Comment by vijay6696** [comment]
yes i have and when sometime i try to restart the application it is showing the courses, but still when i try to download it it is in the "Building the course data" i tried skipping the attachment in the settings but still no progress.
